.travis.yml

```diff
@@ -10,19 +10,33 @@ before_script:
 
 script: tox
 
+install: true
```

Comment:
FYI: this overrides the default `pip install -r requirements.txt` that we don't want Travis to execute because we're using tox anyways.